core/dvector.h

```diff
@@ -630,13 +595,13 @@ template <class T>
 void PoolVector<T>::invert() {
 	T temp;
 	Write w = write();
-	int s = size();
-	int half_s = s / 2;
+	int len = size();
+	int halfLen = len / 2;
```

Comment:
Should't this be snake_cased?

Reply:
Right you are, I'm just used to using camel case.
I've updated it to snake_case.